tests/dbsclient_t/unittests/DBSClientBlockWriter_t.py

```diff
@@ -6,8 +6,8 @@
 import time
 import uuid
 import unittest
-from dbs.apis.dbsClient import *
-from dbs.exceptions.dbsClientException import dbsClientException
+from dbsClient.apis.dbsClient import *
+from dbsClient.exceptions.dbsClientException import dbsClientException
 from RestClient.ErrorHandling.RestClientExceptions import HTTPError
 from ctypes import *
```

Comment:
why not py3 json?

Reply:
json is the integrated python3 library to read/write/... json